operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Operator.java

```diff
@@ -21,7 +21,7 @@
 import io.javaoperatorsdk.operator.processing.LifecycleAware;
 
 @SuppressWarnings("rawtypes")
-public class Operator implements AutoCloseable, LifecycleAware {
```

Comment:
@metacosm this Autoclosable is a little bit confusing here since start is not blocking, removed it, pls let me know what do you think.

Reply:
That change will need to be documented because people might be relying on it.

Reply:
You mean in migration guide?  We don't document changes from v1 -> v2 , just there